src/lib.rs

```diff
@@ -81,7 +81,8 @@ impl Heading<'_> {
     ///
     /// This is calculated in the same way that GitHub calculates it.
     pub fn anchor(&self) -> String {
-        regex!(r"[^\w\- ]")
+        Regex::new(r"[^\w\- ]")
+            .unwrap()
```

Comment:
This will recompile the regex every time this function is called which is unnecessary. We need a lazy static for this, which is what the `regex-macro` crate was doing. If we remove the `regex-macro` dependency we still need to then use `once_cell` or `lazy_static`. Once the following is stabilized we will be able to do it with the standard library: https://doc.rust-lang.org/stable/std/sync/struct.LazyLock.html

Reply:
Oh yeah good catch ... I have dropped the 2nd commit from this PR. I'll address the github anchor calculation in a followup PR.

Reply:
Ok nevermind, I added the commit back with `once_cell` as you suggested.